detail.html

```diff
@@ -0,0 +1,35 @@
+<%inherit file="app_base.htm" />
+
+<%block name="middle">
+    <div class="text-center">
+        <h2>${ name }</h2>
+    </div>
+    <div id="prodPage">
+        <div id="prodPics">
+            <ul>
+                <%iCount = 0 %>
+                % for i in product.image_urls():
+                <%iCount = iCount + 1 %>
```

Comment:
You can make this iCount += 1 and it will do the same thing

Reply:
I had that written at first, but I like the "= var + 1" better because it seems more clear to me. Still a beginner coder.